packages/ag-charts-community/src/chart/series/series.ts

```diff
@@ -775,4 +782,53 @@ export abstract class Series<
         }
         return defaultFormatter(params.value);
     }
+
+    protected getMarkerStyle<TParams>(
+        marker: ISeriesMarker<TDatum, TParams>,
+        params: TParams & Omit<AgSeriesMarkerFormatterParams<TDatum>, 'seriesId'>,
+        defaultStyle: AgSeriesMarkerStyle = marker.getStyle()
+    ) {
+        const defaultSize = { size: params.datum.point?.size ?? 0 };
+        const markerStyle = mergeDefaults(defaultSize, defaultStyle);
+        if (marker.formatter) {
+            const style = this.ctx.callbackCache.call(marker.formatter, {
+                seriesId: this.id,
+                ...markerStyle,
+                ...params,
+            });
+            return mergeDefaults(style, markerStyle);
+        }
+        return markerStyle;
+    }
+
+    protected updateMarkerStyle<TParams>(
+        markerNode: Marker,
+        marker: ISeriesMarker<TDatum, TParams>,
+        params: TParams & Omit<AgSeriesMarkerFormatterParams<TDatum>, 'seriesId'>,
+        defaultStyle: AgSeriesMarkerStyle = marker.getStyle()
+    ) {
+        const { point } = params.datum;
+        const activeStyle = this.getMarkerStyle(marker, params, defaultStyle);
+
+        markerNode.setProperties({
+            visible: this.visible && activeStyle.size > 0 && point && !isNaN(point.x) && !isNaN(point.y),
+            translationX: point?.x,
+            translationY: point?.y,
```

Comment:
Should these be present in this call, given the guard in the following statement?